src/libs/freezeScreenWithLazyLoading.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import FreezeWrapper from './Navigation/FreezeWrapper';
+
+function FrozenComponent<TProps extends React.JSX.IntrinsicAttributes>(InnerComponent: React.ComponentType<TProps>) {
```

Comment:
I think this is now a function returning a React component, so it can be named using small letter
`frozenComponent` and since the function is called `freezeScreen` then maybe call this frozenScreen instead to be consistent.